src/v3/test/integration/authenticator-expired-password.test.tsx

```diff
@@ -68,4 +61,99 @@ describe('authenticator-expired-password', () => {
       },
     );
   });
+
+  it('should not make network request when only new password has a value', async () => {
+    const {
+      authClient, user, findByTestId, findByText,
+    } = await setup({ mockResponse });
+
+    await findByText(/Your password has expired/);
+    await findByText(/Password requirements/);
+
+    const submitButton = await findByTestId('#/properties/submit');
+    const newPasswordEle = await findByTestId('credentials.passcode') as HTMLInputElement;
+
+    await user.type(newPasswordEle, 'superSecretP@ssword12');
+
+    await user.click(submitButton);
+    await findByText(/This field cannot be left blank/);
```

Comment:
be specific this field message comes from confirmPassword field

Reply:
in this case, there can only be one error message, if there are multiple with this text jest will throw an error.

Reply:
but it's not clear which field this error message belongs to, right?

Reply:
agreed @shuowu-okta I think it would be good to scope this `find*` selector within the specific input we care about rather than the global `screen` selectors

Reply:
The test description does, but I will update. :)

Reply:
It's not the same, think about if this error message appears in new password field, test will still pass, but that will be a bug in widget

Reply:
Yep, forgot this is an integration test and should guard against other issues like this. Updated.